src/test/resources/topic-response-driver-deliveries.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "A private topic",
+  "id": 54321,
+  "category_id": 12
+}
```

Comment:
Could this have been done in the test case with setGetResponseData() instead of creating another file?